app/helpers/export_helper.rb

```diff
@@ -31,4 +33,139 @@ def export_excel(project)
 
     workbook
   end
+
+  def export_xccdf(project)
+    Zip::OutputStream.write_buffer do |zio|
+      project.components.eager_load(rules: %i[disa_rule_descriptions checks
+                                              satisfies satisfied_by]).each do |component|
+        version_revision = "V#{component[:release_version]}R#{component[:release_revision]}"
+        file_name = "U_#{component[:name]}_STIG_Readiness_Guide_#{version_revision}-xccdf.xml"
+        zio.put_next_entry(file_name)
+
+        doc = Ox::Document.new
+
+        # Document Headers
+        instruct_xml = Ox::Instruct.new(:xml)
+        instruct_xml['version'] = '1.0'
+        instruct_xml['encoding'] = 'UTF-8'
+        doc << instruct_xml
+
+        instruct_xml_s = Ox::Instruct.new(:'xml-stylesheet')
+        instruct_xml_s['type'] = 'text/xsl'
+        instruct_xml_s['href'] = 'STIG_unclass.xsl'
+        doc << instruct_xml_s
+
+        # Root Benchmark element
+        benchmark = Ox::Element.new('Benchmark')
+        benchmark['xmlns:dc'] = 'http://purl.org/dc/elements/1.1/'
+        benchmark['xmlns:xsi'] = 'http://www.w3.org/2001/XMLSchema-instance'
+        benchmark['xmlns:cpe'] = 'http://cpe.mitre.org/language/2.0'
+        benchmark['xmlns:xhtml'] = 'http://www.w3.org/1999/xhtml'
+        benchmark['xmlns:dsig'] = 'http://www.w3.org/2000/09/xmldsig#'
+        benchmark['xsi:schemaLocation'] = 'http://checklists.nist.gov/xccdf/1.1 ' \
+                                          'http://nvd.nist.gov/schema/xccdf-1.1.4.xsd' \
+                                          'http://cpe.mitre.org/dictionary/2.0 '\
+                                          'http://cpe.mitre.org/files/cpe-dictionary_2.1.xsd'
+        benchmark['id'] = 'Active_Directory_Domain'
```

Comment:
Update to match title

Reply:
https://github.com/mitre/vulcan/pull/342/commits/f17d7512563fdd4638abbd7ecbac295236e2f524